### ISSUE TITLE
CD: Dump JS routing

### ISSUE THREAD

**Issue by eternoendless** [PrestaShop/PrestaShop#21200]
#### Is your feature request related to a problem?

Some actions are performed too late in the build process, resulting in delays. We can leverage CI to automate these actions nightly.

#### Describe the solution you'd like

Dump the JS routing to ensure it's always up to date, then [create an automated Pull Request](https://peterevans.dev/posts/github-actions-how-to-create-pull-requests-automatically/) using the @prestonBot user.

```
php bin/console fos:js-routing:dump --format=json --target=admin-dev/themes/new-theme/js/fos_js_routes.json
```

**Comment by Progi1984** [comment]
Merry Christmas :santa: @eternoendless : https://github.com/PrestaShop/PrestaShop/pull/22588

**Comment by Progi1984** [comment]
Need check before closing

**Comment by Progi1984** [comment]
Closed with #22861 (a PR is created if the file is different, and the CI doesn't create a PR if a PR exists).
